### DISCUSSION TITLE
resize font only in one window

### DISCUSSION THREAD

**Discussion by piotr-sikora-v** [kovidgoyal/kitty#7630] [Q&A]
How can I change font size (cmd+plus) only in one window when I using single-instance?
Now when I changing font size in one window all other is changed :/

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf/#font-sizes

On Sat, Jul 13, 2024 at 07:17:24PM -0700, Piotr Sikora wrote:
> How can I change font size (cmd+plus) only in one window when I using single-instance?
> Now when I changing font size in one window all other is changed :/
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/7630
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>

--
_____________________________________

Dr. Kovid Goyal
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by piotr-sikora-v** [reply]
I have only problem when I use kitty when it start with single instance mode `kitty -1`
When I run separet instance then I can change size of fonts independent to other kitty windows.

I use MacOS

**Reply by kovidgoyal** [reply]
Read the text at the link I posted, pay attention to the word all there.

**Reply by piotr-sikora-v** [reply]
Yes I readed this.
Here is my config:
```
map ctrl+shift+equal change_font_size all +2.0
map ctrl+shift+plus change_font_size all +2.0
map ctrl+equal change_font_size current +2.0
map ctrl+plus change_font_size current +2.0
```
all of them changing all font size in all windows when I start kitty in single instance mode.

**Comment by kovidgoyal** [comment]
Does not reproduce for me. Steps I tried:

kitty --single-instance -o 'map f1 change_font_size current +2.0'
kitty --single-instance
Press f1, font chages only in the currently focused window.

**Comment by piotr-sikora-v** [comment]
Strange.... I just uninstalled kitty and installed agin and it works.
